Makefile

```
@@ -52,7 +52,7 @@ build: clean $(OUT_DIR)/$(PROGRAM_NAME)
 
 update-go-deps:
 	@for m in $$(go list -mod=readonly -m -f '{{ if and (not .Indirect) (not .Main)}}{{.Path}}{{end}}' all); do \
-		go get -d $$m; \
+		go get $$m; \
```

given the name of the target, it seems like it was meant to be
```suggestion
		go get -u $$m; \
```
instead